fe/fe-core/src/main/java/org/apache/doris/common/util/BrokerUtil.java

```diff
@@ -119,6 +119,12 @@ public static List<String> parseColumnsFromPath(
         if (columnsFromPath == null || columnsFromPath.isEmpty()) {
             return Collections.emptyList();
         }
+        if (!caseSensitive) {
```

Comment:
why not add some case in UT or regression test?

Reply:
Will add some test cases.